Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabUiHandler.h

```diff
@@ -73,6 +73,7 @@ namespace AzToolsFramework
         int m_prefabCapsuleRadius = 6;
         int m_prefabBorderThickness = 2;
         int m_prefabFileNameFontSize = 10;
+        int m_prefabEditIconSize = 16;
```

Comment:
does this need to be a value seems like this could just be a constexpr constant?

Reply:
The Procedural Prefab Ui Handler alters some of those values, so I left them all accessible for similar use cases.

Reply:
maybe you could make it a trait type configuration? I think this is fine as it then 
```
struct {
   int PrefableEditIconSize = 16;
} DefaultStyle


class A{
 using Style = DefaultStyle;

   void function() {
      int iconSize = Style::PrefableEditIconSize;

  }
}
```